apps/core/components/ItineraryCard/TripSector.js

```diff
@@ -67,6 +57,31 @@ export default function TripSegment({
   );
 }
 
+export default createFragmentContainer(
+  TripSector,
+  graphql`
+    fragment TripSector on Sector {
+      duration
+      segments {
+        arrivalTime {
+          local
+        }
+        departureTime {
+          local
+        }
```

Comment:
You could use another fragment for this, so you don't have to repeat this logic

Reply:
Could you explain it more, please? I don't know what exactly you mean.

Reply:
I can show you on monday 😊